pkgs/applications/misc/keepassx/community.nix

```diff
@@ -1,28 +1,51 @@
 { stdenv, fetchFromGitHub, cmake, makeWrapper, qttools
-, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, libgpgerror, glibcLocales, libyubikey, yubikey-personalization, libXi, qtx11extras
-, withKeePassHTTP ? true
+
+, curl
+, libargon2
+, libgcrypt
+, libsodium
+, zlib
+, libmicrohttpd
+, libXtst
+, qtbase
+, libgpgerror
+, glibcLocales
+, libyubikey
+, yubikey-personalization
+, libXi
+, qtx11extras
+
+, withKeePassBrowser ? true
+, withKeePassHTTP ? false
+, withKeePassNetworking ? false
+, withKeePassSSHAgent ? false
```

Comment:
You would not enable this?

Reply:
I don't use the feature, so I don't have much of an opinion on this. I went with the [default](https://github.com/keepassxreboot/keepassxc/blob/a06e85f6ceac39851ca694464f74c927dd953f1e/CMakeLists.txt#L50), but I'm happy to change it if it's generally useful to others.

Reply:
The default is not every useful. It also disable `WITH_XC_BROWSER`.